saleor/warehouse/management.py

```diff
@@ -266,9 +280,12 @@ def decrease_stock(order_lines_info: Iterable["OrderLineData"]):
             "quantity_allocated__sum"
         ]
 
-    _decrease_stocks_quantity(
-        order_lines_info, variant_and_warehouse_to_stock, quantity_allocation_for_stocks
-    )
+    if update_stocks:
```

Comment:
Maybe a comment would be useful to explain in what cases we want to enter this logic?

Reply:
Added a docstring about what this param does